docs/public/analytics.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable no-undef */
+

```

Comment:
we should remove docs folder and all references of it. we should just use README for instructions.

Reply:
done